src/execution/expression_executor/execute_operator.cpp

```diff
@@ -111,6 +111,38 @@ void ExpressionExecutor::Execute(const BoundOperatorExpression &expr, Expression
 		} else if (count == 1) {
 			result.SetVectorType(VectorType::CONSTANT_VECTOR);
 		}
+	} else if (expression_type == ExpressionType::OPERATOR_TRY) {
+		auto &child_state = *state->child_states[0];
+		try {
+			Execute(*expr.children[0], &child_state, sel, count, result);
+			return;
+		} catch (std::exception &ex) {
+			ErrorData error(ex);
+			auto error_type = error.Type();
+			if (!Exception::IsExecutionError(error_type)) {
+				throw;
+			}
+		}
+		SelectionVector selvec(1);
+		Vector intermediate(result.GetType(), 1);
+		for (idx_t i = 0; i < count; i++) {
+			selvec.set_index(0, sel ? sel->get_index(i) : i);
+			Value val;
```

Comment:
Does this not need to have an explicit type for the Value?
The `SetValue` could throw an error if the types don't match